README.MD

```diff
@@ -112,6 +88,29 @@ As part of the MCT library, we have a set of example networks on image classific
 
 For more results please see [1]
 
+### Pytorch
+We quantized classification networks from the torchvision library. 
+In the following table we present the ImageNet validation results for these models:
+
+| Network Name              | Float Accuracy  | 8Bit Accuracy   | 
+| --------------------------| ---------------:| ---------------:| 
+| MobileNet V2 [3]          | 71.886          | 71.804          |                                      
```

Comment:
Please show only results on:MobileNet V2 [3], SqueezeNet 1.1 [3] and ResNet-18 [3]